rpc/core/src/wasm/message.rs

```diff
@@ -0,0 +1,1400 @@
+#![allow(non_snake_case)]
+
+use crate::error::RpcError as Error;
+use crate::error::RpcResult as Result;
+use crate::model::*;
+use kaspa_addresses::Address;
+use kaspa_addresses::AddressOrStringArrayT;
+use kaspa_consensus_client::Transaction;
+use kaspa_consensus_client::UtxoEntryReference;
+use kaspa_rpc_macros::declare_typescript_wasm_interface as declare;
+pub use serde_wasm_bindgen::from_value;
+use wasm_bindgen::prelude::*;
+use workflow_wasm::convert::*;
+use workflow_wasm::extensions::*;
+use workflow_wasm::serde::to_value;
+
+macro_rules! try_from {
+    ($name:ident : $from_type:ty, $to_type:ty, $body:block) => {
+        impl TryFrom<$from_type> for $to_type {
+            type Error = Error;
+            fn try_from($name: $from_type) -> Result<Self> {
+                $body
+            }
+        }
+    };
+}
+
+// ---
+
+#[wasm_bindgen(typescript_custom_section)]
+const TS_ACCEPTED_TRANSACTION_IDS: &'static str = r#"
+    /**
+     * Accepted transaction IDs.
+     * 
+     * @category Node RPC
+     */
+    export interface IAcceptedTransactionIds {
+        acceptingBlockHash : HexString;
+        acceptedTransactionIds : HexString[];
+    }
+"#;
+
+#[wasm_bindgen(typescript_custom_section)]
+const TS_UTXO_BY_ADDRESSES_ENTRY: &'static str = r#"
+    /**
+     * Collection of data returned by the GetUtxosByAddresses method.
```

Comment:
How did you determine that this is "unused"?

Reply:
@aspect

Reply:
The reason you don't see the use of the interface is because functions that use it are dynamically generated by macros during the compilation like so: https://github.com/aspectron/rusty-kaspa/blob/typescript/rpc/macros/src/wrpc/wasm.rs#L60

The macro is given a list of API calls https://github.com/aspectron/rusty-kaspa/blob/typescript/rpc/wrpc/wasm/src/client.rs#L931 and for each identifier it generates a function with a matching snake case name as well as corresponding Request and Response types.

Reply:
Yep, this is a good catch.  To optimize data processing, in WASM we convert `RpcUtxosByAddressesEntry` directly to `UtxoEntryReference` and as such `IUtxoByAddressEntry` is not required.  Removed in https://github.com/aspectron/rusty-kaspa/commit/153b1af376f6c43a86ff5c10521c52e2cb87abce